packages/cli/src/cli/localizer/explicit.ts

```diff
@@ -51,18 +52,28 @@ export default function createExplicitLocalizer(
         apiKeyName: "GOOGLE_API_KEY",
         baseUrl: provider.baseUrl,
       });
+    case "ollama":
+      return createAiSdkLocalizer({
+        factory: (_params) => createOllama().languageModel(provider.model),
+        id: provider.id,
+        prompt: provider.prompt,
+        authRequired: false,
+      });
   }
 }
 
 function createAiSdkLocalizer(params: {
-  factory: (params: { apiKey: string; baseUrl?: string }) => LanguageModel;
+  factory: (params: { apiKey?: string; baseUrl?: string }) => LanguageModel;
   id: NonNullable<I18nConfig["provider"]>["id"];
   prompt: string;
-  apiKeyName: string;
+  apiKeyName?: string;
   baseUrl?: string;
+  authRequired?: boolean;
```

Comment:
let's invert and do `skipAuth?: boolean` instead: it's more conventional, and it'd be harded to make mistakes for someone who's not familiar with the codebase v much